### PR TITLE
Find and extract URLs in the report HTML

### PR DESCRIPTION
**Pull request by mfalesni**
... so you can just click and go.

{{pytest: cfme/tests/automate/test_vmware_methods.py -v -k hotadd_disk}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9fa702ebafd17528e6828926c6353e9bac60789a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a6f62d85a0d0f0fe54e903b8cdb7d3b9d9f42dbd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: artifactor/plugins/reporter.py [hunk 1]
```diff
@@ -37,6 +39,11 @@
     '_duration': 0
 }
 
+URL = re.compile(
```
Comment (seandst) [inline artifactor/plugins/reporter.py]: I do not want to parse regex with my eyes.

What does this do?

Comment (mfalesni) [inline artifactor/plugins/reporter.py]: I found that somewhere on the web, this should find URLs in the strings.

Comment (seandst) [inline artifactor/plugins/reporter.py]: I think it definitely needs a comment to explain what it's for, and if the link seems durable, then linking to the source would likely be valuable to the lucky person that wants to debug this.

Comment (mfalesni) [inline artifactor/plugins/reporter.py]: I added the comment also with the url I found the regexp on.

**Comment by dajoRH** [comment]
Lint report for commit f1b63676257758bdf0ce0812fff11132358315c7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**